carmin-server/server/resources/execution.py

```diff
@@ -19,8 +20,28 @@ def get(self, user, db_session, execution_identifier):
             return ErrorCodeAndMessageFormatter(EXECUTION_NOT_FOUND)
         return execution
 
-    def put(self, user, execution_identifier):
-        pass
+    @login_required
+    @get_db_session
+    @unmarshal_request(ExecutionSchema(), partial=True)
+    @marshal_response()
+    def put(self, model, execution_identifier, user, db_session):
```

Comment:
Please replace put with PATCH. We will also change this in the OpenAPI 3.0 descriptor to reflect the changes discussed with @glatard. I tested it and patch works with flask-restful.

Reply:
I didn't want to do this before the change was approved in https://github.com/fli-iam/CARMIN. The issue has been created (https://github.com/fli-iam/CARMIN/issues/86), but I don't know of a pull request that was merged. @glatard should we wait before a PR is merged before implementing it as `PATCH`?